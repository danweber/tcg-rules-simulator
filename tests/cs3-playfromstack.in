VERSION:2


# [All Turns] When this Monster would leave the battle area other than by one of your effects, from this Monster's evolution cards, play 1 Monster card.",

FIELD1:Bird,CS3-ChirpFlare,CS3-02,CS3-02,CS3-01
FIELD2:Shield,CS1-StoneShield,REST

GAME:P1 T3 MAIN 8
DUMP:HAND,DP

ECHO: ----- this may fail if we haven't loaded <dodge>
ECHO: ---- attack in, mon gets played out

CMD:ATTACK Bird Shield
STEP:AUTOANSWER:CAPTURE

DUMP:HAND,DP

EXIT
ECHO: -- choose to evo
STEP:TO:0
EFFECT-TREE
GET-CHOICES
DUMP

EXIT

ECHO: -- pick growl 
STEP:TO:0
EFFECT-TREE
GET-CHOICES
DUMP

