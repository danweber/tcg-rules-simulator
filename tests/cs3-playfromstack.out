FIELD1:Bird,CS3-Chirpflare,CS3-Fairyfeatherlink,CS3-Fairyfeatherlink,CS3-Beaklet,2000DP
FIELD2:Shield,CS1-04,REST,4500DP
HAND1:
HAND2:
GAME:P1 T3 MAIN 8
ECHO: ----- this may fail if we haven't loaded <dodge>
ECHO: ---- attack in, mon gets played out
CAPTURE:Beaklet [All Turns] When this Monster would leave the battle area other than by one of your effects,â€¦
CAPTURE:ChirpFlare When this Monster would leave the battle area, you may suspend it to prevent that removal.
CAPTURE:Target FairyFeatherlink BATTLE-CS3-02
CAPTURE:Target FairyFeatherlink BATTLE-CS3-02
FIELD1:I3,CS3-Fairyfeatherlink,1000DP
FIELD2:Shield,CS1-04,REST,4500DP
HAND1:
HAND2:
GAME:P1 T3 MAIN 5
